docs/DeveloperGuide.adoc

```diff
@@ -798,62 +867,210 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |user |add player’s position |easily pick my lineup for match
+
+|`* * *` |user |delete player |remove him if I kick him out from the team or he quits
+
+|`* * *` |user |edit player’s allocated team |move players between teams
+
+|`* * *` |user |edit player’s contact number |
+
+|`* * *` |user |edit player’s email |
+
+|`* * *` |user |edit player’s jersey number |
+
+|`* * *` |user |edit player’s number of demerits (Y/R cards) |fine/bench the player easily
+
+|`* * *` |user |find a player by name |locate a person without going through the whole list
+
+|`* * *` |new user |see a list of instructions available |navigate and use the application with
+ease
 
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |user |add player remarks |for self note
+
+
+|`* *` |user |add player’s avatar |for facial recognition
+
+|`* *` |user |add player’s match stats (e.g. goals scored) |decide the lineup, give award
+
+|`* *` |user |autofill command |perform task quickly
+
+|`* *` |user |edit player remarks |for self note
+
+|`* *` |user |edit player’s address |
+
+|`* *` |user |edit player’s avatar |
+
+|`* *` |user |edit player’s lineup status |organise the lineup (e.g. main team, reserves)
+
+|`* *` |user |edit player’s match stats |decide the lineup, give award
+
+|`* *` |user |edit player’s position |easily pick my lineup for match
+
+|`* *` |user |find player by team name |locate a team without going through the whole list
+
+|`* *` |user |password login |If user wants to protect certain information
+
+|`* *` |user |submit feedback to developers |developers can improve the application constantly
+
+|`*` |user |edit player’s name |
+
+|`*` |user with many players in the team managing application |sort by player name |locate a person easily
+
+|`*` |user |sort by team name |identify group lineup easily
+
+
 |=======================================================================
 
-_{More to be added}_
+
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `MTM` and the *Team Manager* is the `user`, unless specified otherwise)
+
+[discrete]
+=== Use case: Add player
+
+*MSS*
+
+1.  User enter command to add
+2.  MTM prompt user of format to enter player's name and details
+3.  User enter player's name and details in required format
+4.  User press enter to insert person into storage
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 4a. User did not enter any details.
++
+[none]
+** 4a1. MTM inform user that it is an invalid add command.
++
+Use case ends.
+
+[discrete]
+=== Use case: Edit player's team
+
+*MSS*
+
+1.  User enter command to find
```

Comment:
For your MSS, do try to ensure it's more of like a back and forth interaction between the system and the user. So try to model and present your use cases in such a way that the system responds right after a user does something and so on and so forth.
In this case, you can write something like:
1. User enters command to find player's name.
2. MTM displays the list of players found.
3. ...